src/db/utils/jobs/migration/ConvertTickDateClimbedFromStringToDate.ts

```diff
@@ -0,0 +1,30 @@
+import { connectDB } from '../../../index.js'
+import { logger } from '../../../../logger.js'
+import { getTickModel } from '../../../TickSchema.js'
+
+/**
+ * Converts dateClimbed field on ticks from string to date.
+ */
+const onConnected = async (): Promise<void> => {
+  logger.info('Migrating ticks...')
+  const tickModel = getTickModel()
+
+  await (await tickModel.updateMany(
+    { dateClimbed: { $exists: true } },
+    {
+      $set: {
+        dateClimbed: {
+          $dateFromString: {
+            dateString: '$dateClimbed',
+            // We want to ascribe an hour of day to the climb, so it shows
+            // up on the correct day when displayed in the user's timezone.
+            // Most climbs are in the US, MT time is a good first approximation.
+            timezone: 'America/Denver'
```

Comment:
I've not written a Mongo migration before, so please check that this is correct!

Reply:
I think it's ok to assume Mountain time.  This feature continues to evolve.  Since we have the users MP Id, we can re-import on their behalf, or ask the users to initiate the re-import.